### DISCUSSION TITLE
WiX 5.0.0: heat.exe reg

### DISCUSSION THREAD

**Discussion by rowbot38** [wixtoolset/issues#8128] [Questions]
With 5.0, how do we harvest .reg files? With 3.14, we use "heat.exe reg.."

**Comment by rowbot38** [comment]
Anyone know?

**Reply by bevanweiss** [reply]
I think the recommendation is that .reg files should be manually configured.
There's probably a level of automation that could be done.. but nothing that i'm aware of within WiX v5 for this.

I'm sure if you put forward a PR that added such a feature in a reasonable manner it would be accepted.

**Reply by bevanweiss** [reply]
Or you could look into the FireGiant HeatWave Advanced Harvesting options
https://docs.firegiant.com/heatwave/build-tools/harvesting/comparisons/
